mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -61,10 +68,13 @@ CLASS net/minecraft/class_1752 net/minecraft/entity/LivingEntity
 	METHOD method_7160 endCombat ()V
 	METHOD method_7161 damageArmor (F)V
 		ARG 1 value
+	METHOD method_7163 isBaby ()Z
```

Comment:
this doesnt feel like a yarn name. try isChild

Reply:
took it from nbt

Reply:
it's a yarn name bruh